<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1566577060613" stop="1566577068702" version="1.5.4" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default Suite : home</name>
    <title>Default Suite : home</title>
    <test-cases>
        <test-case start="1566577067510" stop="1566577068695" status="failed">
            <name>checkForUserEmail</name>
            <failure>
                <message>AssertionError: expected [michael.lawson@reqres.in] but found [null]</message>
                <stack-trace>java.lang.AssertionError: expected [michael.lawson@reqres.in] but found [null]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at Tests.Test_MainPage.checkForUserEmail(Test_MainPage.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default Suite"/>
                <label name="testGroup" value="home"/>
                <label name="testClass" value="Tests.Test_MainPage"/>
                <label name="testMethod" value="checkForUserEmail"/>
                <label name="host" value="W16_RYBCHENKO"/>
                <label name="thread" value="9284@W16_RYBCHENKO.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1566577068696" stop="1566577068698" status="passed">
            <name>checkStatus</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default Suite"/>
                <label name="testGroup" value="home"/>
                <label name="testClass" value="Tests.Test_MainPage"/>
                <label name="testMethod" value="checkStatus"/>
                <label name="host" value="W16_RYBCHENKO"/>
                <label name="thread" value="9284@W16_RYBCHENKO.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
